services/painting-process-equipment-defect-detection-model-service/app/dependencies.py

```
@@ -0,0 +1,81 @@
+import joblib
+import shap
+import yaml
+import os
```

Ruff (0.12.2)

`os` imported but unused

Remove unused import: `os`

(F401)